WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -420,6 +436,60 @@ local methods = {
             end
         end
 
+        function self.callbacks.OnDragStart(_, _)
```

Comment:
Can you make this function hide various bits of the frame while dragging? E.g. the visibility icon, the ungroup, the move up/down icons.